test/fuzz/simplify.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Halide.h"
 #include <array>
+#include <fuzzer/FuzzedDataProvider.h>
```

Comment:
What happens if libfuzzer isn't installed on the build machine -- will this test be able to compile?

Reply:
Libfuzzer is shipped with clang as part of the llvm project as clang/llvm are already dependencies of this project I don't think that this will be an issue. That specific [header file](https://github.com/llvm/llvm-project/blob/main/compiler-rt/include/fuzzer/FuzzedDataProvider.h) is sort of fuzzer adjacent, and is more of a helper for transforming raw data into structured data. But the header itself is not required to write a fuzzer.